app/controllers/api/v1/users/registrations_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Api::V1::Users
+  class RegistrationsController < ApiController
+    def create
+      Api::V1::Users::Registrations::Operation::Create.call(params.permit!.to_h) do |result|
```

Comment:
Please describe for what `params.permit!.to_h`???

Reply:
As i found Operation do not support ActionController::Parameters object. And for using to_h method we have to permit parameters.